src/Server.php

```diff
@@ -53,11 +53,11 @@ class Server
 
     /**
      * Create Server instance.
-     * @param FilesystemInterface $source The source file system.
-     * @param FilesystemInterface $cache  The cache file system.
-     * @param ApiInterface        $api    The image manipulation API.
+     * @param FilesystemInterface      $source The source file system.
+     * @param FilesystemInterface|null $cache  The cache file system.
+     * @param ApiInterface             $api    The image manipulation API.
      */
-    public function __construct(FilesystemInterface $source, FilesystemInterface $cache, ApiInterface $api)
+    public function __construct(FilesystemInterface $source, $cache, ApiInterface $api)
```

Comment:
Only problem is this breaks [PSR-2 compliance](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md#44-method-arguments), which might break build steps. This is annoying. Might have to just go back to the way you had it.